.github/workflows/mayhem.yml

```diff
@@ -225,31 +225,19 @@ jobs:
         sudo -u $USER mdsbom scout ${{ env.REGISTRY }}/${{ env.REPO_NAME }}/car:${{ github.sha }} --sca-report-out dsbom-car.sarif --workspace ${{ env.WORKSPACE }}
         sudo -u $USER mdsbom scout ${{ env.REGISTRY }}/${{ env.REPO_NAME }}/api:${{ github.sha }} --sca-report-out dsbom-api.sarif --workspace ${{ env.WORKSPACE }}
 
-    - name: Patch docker scout SARIF report for CodeQL
-      run : |
-        jq 'walk(if type == "object" and has("startLine") and .startLine == 0 then .startLine = 1 else . end |
-            if type == "object" and has("endLine") and .endLine == 0 then .endLine = 1 else . end |
-            if type == "object" and has("endColumn") and .endColumn == 0 then .endColumn = 1 else . end |
-            if type == "object" and has("startColumn") and .startColumn == 0 then .startColumn = 1 else . end)' dsbom-api.sarif > dsbom-api-patched.sarif
-
-        jq 'walk(if type == "object" and has("startLine") and .startLine == 0 then .startLine = 1 else . end |
-            if type == "object" and has("endLine") and .endLine == 0 then .endLine = 1 else . end |
-            if type == "object" and has("endColumn") and .endColumn == 0 then .endColumn = 1 else . end |
-            if type == "object" and has("startColumn") and .startColumn == 0 then .startColumn = 1 else . end)' dsbom-car.sarif > dsbom-car-patched.sarif
-
     - name: Upload SARIF file
       id: upload-api
       uses: github/codeql-action/upload-sarif@v3
       with:
-        sarif_file: dsbom-api-patched.sarif
+        sarif_file: dsbom-car.sarif
```

Comment:
@GeorgeLS I don't think I can edit on your repo, can you change these two uploads to `api` instead of `car`